src/core/searchUtils.js

```diff
@@ -57,10 +59,16 @@ export function addVersionCompatibilityToFilters({
     // with a version of at least 57, at least for now. Find the explanation
     // here: https://github.com/mozilla/addons-frontend/pull/2969#issuecomment-323551742
     if (browserVersion >= 57) {
-      log.debug(oneLine`Setting "compatibleWithVersion" to current application
+      if (config.get('checkForVersionCompatiblityInAPIRequests')) {
+        log.debug(oneLine`Setting "compatibleWithVersion" to current application
         version (Firefox ${browserVersion}) so only relevant extensions are
         displayed.`);
-      newFilters.compatibleWithVersion = userAgentInfo.browser.version;
+        newFilters.compatibleWithVersion = userAgentInfo.browser.version;
+      } else {
+        log.debug(oneLine`checkForVersionCompatiblityInAPIRequests config set;
```

Comment:
I think `log.warn()` would be more helpful here since disabling a production feature (while in development) is risky.